### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by kramstrom**
forgot to update changelog in asgi lifespan PR https://github.com/modal-labs/modal-client/pull/2183

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,7 +21,31 @@ We appreciate your patience while we speedily work towards a stable release of t
...
+        web_app = FastAPI(lifespan=lifespan)
+
+        @web_app.get("/")
+        def get_state(request: Request):
+            return {"message": f"This is the state: {request.state.foo}"}
+
+        return web_app
+    ```
+
+    and enabling support for `gradio>=v4` amongst other libraries using lifespans
```
Comment (mwaskom) [inline CHANGELOG.md:48]: I'm not sure if the false markup here is a problem or not but note that i think we have previously accidentally broken the svelte build by including `>` tokens in the changelog. Maybe the simplest change would be `gradio v4+`?

Comment (kramstrom) [reply CHANGELOG.md:48]: Hm this works locally when building though but happy to change if you think it's safer
Screenshot 2024-09-12 at 15 38 52

Comment (mwaskom) [reply CHANGELOG.md:48]: ah that's good — maybe just an issue with `<` actually https://modal-com.slack.com/archives/C03130Y0AGL/p1722958342726519?thread_ts=1722956785.894109&cid=C03130Y0AGL

Comment (kramstrom) [reply CHANGELOG.md:48]: Seems like it's fine because it's in a code block, so both < and > works if you stick them within ``

Comment (freider) [inline CHANGELOG.md:48]: nit
```suggestion
    which enables support for `gradio>=v4` amongst other libraries using lifespans
```

**Comment by kramstrom** [comment]
@prbot approve

**Comment by freider** [comment]
lgtm

**Pull request merged**